packages/seal-loupe-adapter/src/LoupeSearcher.php

```diff
@@ -122,6 +122,15 @@ public function search(Search $search): Result
         );
     }
 
+    private function escapeFilterValue(string|int|float|bool $value): string
+    {
+        return match (true) {
+            \is_string($value) => '"' . \addslashes($value) . '"',
```

Comment:
`addslashes` sadly not enough still run into:

```
Loupe\Loupe\Exception\FilterFormatException: Col 7: Error: Expected valid string or float value, got '"'
```

The test uses a very special string:

```php
$value = "The 17\" O'Conner && O`Series \n OR a || 1%2 book?";
```

@Toflar do you have a hint for me?

Reply:
I think Loupe should provide escaping itself for the filter. I'll check if I can implement this.

Reply:
If I use '"'  around the string and still escaping the `"` in `17"` to `17\"` does not have effect and fails with:

>  Loupe\Loupe\Exception\FilterFormatException: Col 7: Error: Expected valid string or float value, got '"'

If using `'` it fails in `'O'Conner'` even it is escaped to `'O\'Conner'`:

> Loupe\Loupe\Exception\FilterFormatException: Col 20: Error: Expected filterable attribute, got 'Conner'

Reply:
Yes, the parser is not able to do this atm.

Reply:
I see, keep me uptodate :)

Reply:
Can you test with https://github.com/loupe-php/loupe/pull/54? :)